docs/servicenow.md

```diff
@@ -0,0 +1,149 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+with a ServiceNow object. A list of the ServiceNow fields that have been
+tested when creating and editing tickets will be included. Because each
```

Comment:
Change "will be" to "is". I'll need to make this change across the other tools as well.